user_tools/src/spark_rapids_pytools/resources/collect.sh

```diff
@@ -48,11 +55,17 @@ free -h >> $OUTPUT_NODE_INFO
 
 echo "" >> $OUTPUT_NODE_INFO
 echo "[Network adapter]" >> $OUTPUT_NODE_INFO
+
 if command -v lshw ; then
     lshw -C network >> $OUTPUT_NODE_INFO
 else
-    # Downgrade to 'lspci' on EMR as it's not installed by default
-    /usr/sbin/lspci | grep 'Ethernet controller' >> $OUTPUT_NODE_INFO
+    # Downgrade to 'lspci'
+    if [[ "$PLATFORM_TYPE" == *"databricks"* ]]; then
+        sudo apt install -y pciutils
```

Comment:
Question: is `sudo` required anywhere else in the diagnostic tool?  We may want to be able to fail gracefully if user does not have sudo permissions on that node (which hopefully is rare).

Reply:
Added checks to handle `sudo`